api/src/main/java/org/openmrs/module/fhir/api/impl/PractitionerServiceImpl.java

```diff
@@ -148,4 +159,46 @@ public Practitioner getPractitioner(String id) {
 	private List<Provider> searchProvidersByQuery(String query) {
 		return Context.getProviderService().getProviders(query, null, null, null, false);
 	}
+	
+	/**
+	 * @see org.openmrs.module.fhir.api.PractitionerService#createFHIRPractitioner(String)
+	 */
+	public Practitioner createFHIRPractitioner(Practitioner practitioner) {
+		Provider provider=null;
+		// If the representation has unknown gender and the birthday is null, that means the practitioner is not associated with any person resource. Because those attributes are compulsary if there is a person.
+		if (practitioner.getGender().equalsIgnoreCase(String.valueOf(AdministrativeGenderEnum.UNKNOWN))&&practitioner.getBirthDate()==null){ 
```

Comment:
@sashrika , re. leaving gender blank, I think you tested this via the web UI. If created via the web UI, the person object is validated through the personvalidator, and so, it cant have a blank gender. But we're creating persons via the API, and so the call doesn't go through this validator. If you look at the person table, the gender field can be blank :)